docker-compose.yml

```diff
@@ -1,19 +1,17 @@
 version: '3.3'
 services: 
-  ui-e2e-matdev:
-    image: matdevuserinterface
-    build: .
-    container_name: cymatdev
-    command: "npm run matdev:all:ui:tests"
-
   ui-e2e-dev:
-    image: devuserinterface
+    image: bonniefhir
```

Comment:
why the image name bonnieFhir? is it somehow related to bonnie?

Reply:
I guess the image could be named whatever we want. I named it that because the environment variables that I added are specific to Bonnie FHIR. I was using a single BonnieFHIR test for testing this setup. But I guess once we add all of MAT there as well, will give it a better image name.